pkg/controllers/subnetset/subnetset_webhook.go

```diff
@@ -0,0 +1,112 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package subnetset
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
+)
+
+// log is for logging in this package.
+var subnetsetlog = logf.Log.WithName("subnetset-webhook")
+
+var NSXOperatorSA = "system:serviceaccount:vmware-system-nsx:ncp-svc-account"
```

Comment:
in WCP env, do you check that the req.UserInfo.Username   is the above?

Reply:
I didn't test this recently on WCP env, but in previous test I recall the name is same in WCP env.